R/utils.R

```diff
@@ -167,10 +167,20 @@ ask <- function(question, default = FALSE) {
 
 }
 
-proceed <- function(default = FALSE) {
-  ask("Do you want to proceed?", default = default)
+proceed <- function(prompt = interactive(), default = TRUE) {
+  if (prompt) {
```

Comment:
What is the behavior if the user sets `prompt = TRUE` in a non-interactive session?

Reply:
Hmmm, `ask()` has another layer of fallbacks, so it'll return `default`.